src/corner_kick/src/utils/ros/index.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * This file contains ROS methods to get and set values in the ROS Param Server
+ */
+
+import { Message, Ros, Service, ServiceRequest, Topic } from 'roslib';
+
+const ros: Ros = new Ros({});
+
+/**
+ * Connect to ROS
+ */
+export const connect = (url = 'ws://localhost:9090', timeout = 100) => {
+    return new Promise((resolve, reject) => {
+        ros.connect(url);
+
+        ros.on('connection', () => resolve());
+        ros.on('error', () => reject('Cannot connect'));
+
+        // We reject if the timeout has elapsed
+        setTimeout(() => reject(), timeout);
+    });
+};
+
+/**
+ * Disconnect from ROS
+ */
+export const stop = () => {
+    ros.close();
+};
+
+/**
+ * Subscribe to ROS Topic
+ */
+export const subscribeToROSTopic = (
+    name: string,
+    messageType: string,
+    callback: (message: ROSLIB.Message) => void,
+) => {
+    const topic = new Topic({
+        messageType,
+        name,
+        ros,
+    });
+    topic.subscribe(callback);
+};
+
+/**
+ * Unsubscribe from ROS Topic
```

Comment:
More detailed comment describing each param (using @params, see JSDoc format) would be nice